extensions/amp-analytics/amp-analytics.md

```diff
@@ -669,7 +669,7 @@ In the example below, an `iframe` URL is not specified, and `beacon` and `xhrpos
 }
 ```
 
-To learn more, see [this example that implements iframe transport client API] (https://github.com/ampproject/amphtml/blob/master/examples/analytics-iframe-transport-remote-frame.html) and [this example page that incorporates that iframe](https://github.com/ampproject/amphtml/blob/master/examples/analytics-iframe-transport.amp.html). The example loads a [fake ad](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_iframe_transport.html), which contains the `amp-analytics` tag. Note that the fake ad content includes some extra configuration instructions that must be followed.
```

Comment:
It's not clear in the GitHub UI, but what I did here was remove a space between the [] and the () in the first link. This was causing the URL to display, rather than be a clickable link.